shared/constants/network.ts

```diff
@@ -1012,4 +1012,13 @@ export const TEST_NETWORK_IDS = [
   CHAIN_IDS.SEPOLIA,
   CHAIN_IDS.LINEA_GOERLI,
   CHAIN_IDS.LINEA_SEPOLIA,
+  CHAIN_IDS.ARBITRUM_GOERLI,
```

Comment:
Shouldn't we define the network names as well?

Reply:
Indeed 🤔 Lemme see if we have defined them already somewhere else

Reply:
After discussing this internally, we will remove those for now, to avoid any unwanted side effects.